deploy/k8s/openapi/templates/cm.openapi.yaml

```diff
@@ -1763,18 +1766,18 @@ items:
                   type: string
             responses:
               '201':
-                description: operation queued. Task Id returned.
+                description: URL invalidated correctly
                 content:
                   application/json:
                     schema:
                       $ref: '#/components/schemas/Invalidate-ok'
               '400':
-                description: Error on operation
+                description: Error invalidating URL
                 content:
                   application/json:
                     schema:
-                      $ref: '#/components/schemas/Upload-notok'
-        '/tasks/':
+                      $ref: '#/components/schemas/Task-Creation-Error'
+        '/tasks':
```

Comment:
So this has been fixed? 😮 Now users can make get requests to `<firecrest-url>/tasks`?

Reply:
Yes, it is, so now both options can be use (because `/tasks` allows `/tasks/`)